apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/AbstractRdaLoadJob.java

```diff
@@ -73,6 +76,11 @@ public PipelineJobOutcome call() throws Exception {
       return NOTHING_TO_DO;
     }
     try {
+      try (RdaSource<TResponse, TClaim> source = preJobTask.call();
+          RdaSink<TResponse, TClaim> sink = sinkFactory.call()) {
```

Comment:
The DLQ cleanup task should never use the asynchronus `ConcurrentRdaSink`.  Doing so could allow the DLQ record to be deleted before the claim has actually been updated by a worker thread.